client/db/src/stats.rs

```diff
@@ -97,15 +98,15 @@ impl StateUsageStats {
 			}
 		}
 
-		sp_state_machine::UsageInfo {
+		sp_stats::UsageInfo {
 			reads: unit(&self.reads, &self.bytes_read),
 			writes: unit(&self.writes, &self.bytes_written),
 			cache_reads: unit(&self.reads_cache, &self.bytes_read_cache),
 			// TODO: Proper tracking state of memory footprint here requires
 			//       imposing `MallocSizeOf` requirement on half of the codebase,
 			//       so it is an open question how to do it better
 			memory: 0,
-			started: self.started,
+			started: SystemTime::now().duration_since(UNIX_EPOCH).expect("Time went backwards"),
```

Comment:
but why?

Reply:
This was supposed to be stub code, thanks for catching that!
I've updated the code, we now compute the duration since UNIX_EPOCH and substract the elapsed time :)